flocker/node/agents/functional/test_cinder.py

```diff
@@ -49,7 +49,33 @@
 
 # Tests requiring virsh can currently only be run on a devstack installation
 # that is not within our CI system. This will be addressed with FLOC-2972.
-require_virsh = skipIf(
+#
+# In the meantime, you can do the following (provided you have access):
```

Comment:
> > What's the reason for your preference?

> That's where I look when I want to know how to run tests. We have the acceptance test docs in the contributing docs and this seems similar so I thought it fit into a similar place.

Interesting. I look at rendered documentation as a last resort. Probably mostly because of habit.  Maybe also to avoid the context switch and the inevitable intermediate problem of finding the relevant documentation.